pkg/controller/database/postgres/postgres.go

```diff
@@ -41,18 +40,21 @@ import (
 
 	"github.com/crossplane-contrib/provider-in-cluster/apis/database/v1alpha1"
 	"github.com/crossplane-contrib/provider-in-cluster/pkg/client/database/postgres"
+	"github.com/crossplane-contrib/provider-in-cluster/pkg/controller/utils"
 )
 
 const (
-	errUnexpectedObject                  = "the managed resource is not a Postgres resource"
+	errUnexpectedObject    = "the managed resource is not a Postgres resource" //nolint:golint
+	errDelete              = "failed to delete the Postgres resource"          //nolint:golint
+	errDeploymentMsg       = "failed to get postgres deployment"               //nolint:golint
+	errServiceMsg          = "failed to get postgres service"                  //nolint:golint
+	errPVCCreateMsg        = "failed to create or update postgres PVC"         //nolint:golint
+	errDeployCreateMsg     = "failed to create or update postgres deployment"  //nolint:golint
+	errSVCCreateMsg        = "failed to create or update postgres service"     //nolint:golint
+	errGeneratePasswordMsg = "failed to generate potential postgres password"  //nolint:golint
```

Comment:
just wondering, what does the linter complain about here?

Reply:
I am not sure, it complains that these lines aren't commented, although I think need to be, since none of these and exported.